### ISSUE TITLE
Aspect Map and Shade (Sun Angles) Map

### ISSUE THREAD

**Issue by nthh** [osmandapp/OsmAnd#20849]
### Describe the idea

Ability to see the aspect (N,S,E,W) colored on the map
Could also use this information to generate a shade map that could be filtered by time of day 

[Example here](https://gisgeography.com/aspect-map/)

### Expected behaviour

New features in Topography/Terrain

### Alternatives you've considered

I don't see any option to do this now

**Comment by musoke** [comment]
This would be very useful to me!

**Comment by voggeneder** [comment]
i agree, highly useful for alpine tour planning to avoid avalanche prone slopes.
